extras/userdoc/md/documentation/part-1-neurons-and-simple-neural-networks.md

```diff
@@ -71,11 +71,11 @@ for.
 
     dir(nest)
 
-One such command is `nest.Models()`, which will return a list of all the
+One such command is `nest.Models?`, which will return a list of all the
 available models you can use. If you want to obtain more information about a
-particular command, you may use Python’s standard help system.
+particular command, you may use iPython’s standard help system.
 
-    nest.Models?
+    nest.Models? 
 
```

Comment:
Thanks for changing.
Unfortunately there are still two little problems:
* the i in iPython needs to be upper-case
* there's a trailing whitespace in line 78, which needs to go away.

Sorry for being so niggling about this!

Reply:
No problem, thanks for your help